src/lib/operators/projection.cpp

```diff
@@ -257,7 +257,7 @@ std::shared_ptr<const Table> Projection::_on_execute() {
     auto chunk = std::shared_ptr<Chunk>{};
     if (output_table_type == TableType::Data) {
       chunk = std::make_shared<Chunk>(std::move(output_segments_by_chunk[chunk_id]), input_chunk->mvcc_data());
-      chunk->increase_invalid_row_count(input_chunk->invalid_row_count());
+      chunk->increase_invalid_row_count(input_chunk->invalid_row_count(), std::memory_order_relaxed);
```

Comment:
(again, note memory order not really relevant here?)

Reply:
Well, we allow the CPU to change the order. So it is probably not measurable, but also not irrelevant.

Reply:
Added a comment above the if/else.